builder/iOSTealiumTest/TealiumHelper.swift

```diff
@@ -175,6 +177,27 @@ class TealiumHelper {
         let dispatch = TealiumEvent(title, dataLayer: data)
         tealium?.track(dispatch)
     }
+    
+    func fetchMoments(completion: @escaping (Result<EngineResponse, Error>) -> ()){
+        // example showing all the different types of attributes
+        tealium?.momentsAPI?.fetchEngineResponse(engineID: "<your-engine-id>", completion: { engineResponse in
```

Comment:
Could we have a field here that can be filled from the demo app directly? 
So you wouldn't need to change the code to see what happens (but you could still do).
It's also not clear what happens when you click on the button without setting a correct engineId. 
It just feels as if it's loading. 

Another small thing, the indentation down here is a bit off.